flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/test/java/org/apache/flink/cdc/connectors/mysql/source/MySqlOnLineSchemaMigrationITCase.java

```diff
@@ -310,6 +331,192 @@ public void testGhOstSchemaMigrationFromScratch() throws Exception {
                 fetchResults(events, 2));
     }
 
+    @Test
+    public void testPtOscSchemaMigrationFromScratch() throws Exception {
+        LOG.info("Step 2: Start pipeline job");
+        //        Thread.sleep(5000000);
+        env.setParallelism(1);
+        customerDatabase.createAndInitialize();
+        TableId tableId = TableId.tableId(customerDatabase.getDatabaseName(), "customers_1");
```

Comment:
We may avoid table name conflicts by creating & destroying database in `@Before` and `@After` method for each test case